src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -948,41 +952,55 @@ internal static void CreateActivityListener(DiagnosticSourceEventSource eventSou
                     return false;
                 };
 
-                eventSource._activityListener.ActivityStarted = activity =>
+                eventSource._activityListener.ActivityStarted = activity => OnActivityStarted(eventSource, activity);
+
+                eventSource._activityListener.ActivityStopped = activity => OnActivityStopped(eventSource, activity);
```

Comment:
Copy paste error? Duplicate lines

Reply:
Unless my eyes deceive me one is 'Started' and the other is 'Stopped' : )

Reply:
You're right - ... I'm... well I guess I need to wake up for real.